.github/workflows/release-step-1_manual_create-release-pr.yml

```diff
@@ -1,21 +1,31 @@
-name: One App Release Pull Request
+name: '[Release step 1 | manual] Create Release PR'
 on:
   schedule:
-  - cron: "0 16 * * 3"
+    - cron: "0 16 * * 3"
   repository_dispatch:
     types: [prepare-release-command]
+  workflow_dispatch:
```

Comment:
This `workflow_dispatch` doesn't have any content in it. Is that intentional?

Reply:
It doesn't need to, that just means it can be dispatched from other workflows, but no further config is needed.